me.brokenby.PdfMetadataEditor.yml

```diff
@@ -0,0 +1,38 @@
+app-id: me.brokenby.PdfMetadataEditor
```

Comment:
brokenby.me doesn't seem to resolve. It has to be domain under your control.

Reply:
I think it should be, based on the appstream file, `me.broken_by.*`

Reply:
The domain is broken-by,me , and since it didn't accept the '-' i simply removed it .  Fixed in latest commit.

Reply:
The doc says to replace `-` by `_`.

https://docs.flatpak.org/en/latest/conventions.html#application-ids